src/listener/Link.php

```diff
@@ -14,14 +14,17 @@
  */
 class Link extends InlineListener
 {
+
+    public $wrapper = '<a href="{link}" target="_blank">{text}</a>';
```

Comment:
please add php doc including @since tag.

Reply:
Done! Not sure what else to say there 😕

Reply:
Take a look at the example i just made: https://github.com/nadar/quill-delta-parser/pull/29/files

Reply:
Alright, updated!